src/components/Footer.tsx

```diff
@@ -19,7 +19,7 @@ export function Footer() {
   const currentYear = new Date().getFullYear();
 
   return (
-    <footer className="w-full bg-yaqiin-500 text-yaqiin-100 mt-auto px-4">
+    <footer className="w-full bg-yaqiin-800 text-yaqiin-100 mt-auto px-4">
```

Comment:
The existing background color works well with our current design, I recommend keeping it